### PR TITLE
NEWS: update v2.14.0

### PR DESCRIPTION
**Pull request by prestist**
Working through version bumping ignition [1353](https://github.com/coreos/ignition/issues/1353)

### CODE REVIEW

**Comment by jlebon** [comment]
It'd be good to get the following PRs in before cutting a release:
- https://github.com/coreos/ignition/pull/1350
- https://github.com/coreos/ignition/pull/1319
- https://github.com/coreos/ignition/pull/1352
- https://github.com/coreos/ignition/pull/1264

**Comment by prestist** [comment]
> It'd be good to get the following PRs in before cutting a release:
> 
> * [Delete userdata from VirtualBox/VMware after Ignition completes #1350](https://github.com/coreos/ignition/pull/1350)
> * [internal/exec/stages/disks: prevent races with udev #1319](https://github.com/coreos/ignition/pull/1319)
> * [internal/*: remove enablement symlinks for a unit before disabling #1352](https://github.com/coreos/ignition/pull/1352)
> * [internal/resource: support S3 access point URLs #1264](https://github.com/coreos/ignition/pull/1264)

Okay awesome, I will keep this in a hold position till those land.

**Comment by bgilbert** [comment]
We decided not to hold the release for #1319.

**Comment by bgilbert** [comment]
Would be good to rebase this, to make it clear that the changelog includes entries for commits newer than the current base commit.

**Comment by prestist** [comment]
> Would be good to rebase this, to make it clear that the changelog includes entries for commits newer than the current base commit.

Done!

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,31 @@
+11-May-2022 Ignition 2.14.0
+
+    Features:
```
Comment (bgilbert) [inline NEWS:3]: Incorrect indent.

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,31 @@
+11-May-2022 Ignition 2.14.0
```
Comment (bgilbert) [inline NEWS:1]: Nit: might as well update the date.

File: NEWS [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+
+    Features:
+
+    - Support KubeVirt platform
+    - Support AWS arn: URLs for S3 objects and access points
```
Comment (bgilbert) [inline NEWS:6]: Whoops, this should be `(3.4.0-exp)`.

File: NEWS [hunk 4]
```diff
@@ -1,3 +1,31 @@
...
+    - Support setting setuid/setgid/sticky mode bits (3.4.0-exp)
+    - Warn if setuid/setgid/sticky mode bits specified (3.0.0 - 3.3.0)
+    - Support UEFI Secure Boot on VMware
+    - Add arm64 support to ignition-validate container
+    - Document S3 fetch semantics in operator notes
+
+  Bug Fixes:
+
+    - Fix disabling systemd units with pre-existing enablement symlinks
+    - Fix reuse of statically keyed LUKS volume (2.12.0 regression)
```
Comment (bgilbert) [inline NEWS:24]: Tiny nit: `volumes` for consistency with the pluralization of the other items.

File: NEWS [hunk 5]
```diff
@@ -1,3 +1,31 @@
...
+  Changes:
+
+    - Delete userdata after provisioning on VirtualBox and VMware by default
+      (see operator notes for details)
+    - Support setting setuid/setgid/sticky mode bits (3.4.0-exp)
+    - Warn if setuid/setgid/sticky mode bits specified (3.0.0 - 3.3.0)
+    - Support UEFI Secure Boot on VMware
+    - Add arm64 support to ignition-validate container
+    - Document S3 fetch semantics in operator notes
+
```
Comment (bgilbert) [inline NEWS:20]: Let's add an item for https://github.com/coreos/ignition/pull/1365, maybe something like "Document considerations for handling secrets in operator notes".  This is more general than the userdata deletion parenthetical above; I think there's value in mentioning both.

**Review by bgilbert — COMMENTED** [review]
Please rebase again for #1365.

File: NEWS (hunk 2 above)
Comment (prestist) [reply NEWS]: of course!

File: NEWS (hunk 3 above)
Comment (prestist) [reply NEWS]: Nice catch!

File: NEWS (hunk 4 above)
Comment (prestist) [reply NEWS]: Done!

File: NEWS (hunk 5 above)
Comment (prestist) [reply NEWS:21]: I went with your wording to the T. Thank you for the suggestion

**Review by bgilbert — APPROVED** [review]
:tada:

**Pull request merged**